R/pak.R

```diff
@@ -14,18 +14,20 @@
 #'   packages, just one.
 #'
 #' @param pkg Package names or remote package specifications to install.
-#'   See [pak package sources][pak_package_sources] for details.
+#'   See [pak package sources][pak_package_sources] for details. If `NULL`,
+#'   will install all development dependencies for the current package.
+#' @inheritParams pkg_install
 #' @param ... Extra arguments are passed to [pkg_install()] or
 #'   [local_install_dev_deps()].
 #'
 #' @export
 #' @family package functions
 #' @family local package trees
 
-pak <- function(pkg = NULL, ...) {
+pak <- function(pkg = NULL, upgrade = FALSE, ...) {
```

Comment:
I actually omitted this on purpose, because the default is different now for `pkg_install()` and `local_install_dev_deps()`.